src/tests/functional/plugin/shared/src/behavior/ov_infer_request/infer_request_dynamic.cpp

```diff
@@ -68,6 +69,94 @@ void OVInferRequestDynamicTests::TearDown() {
     function.reset();
 }
 
+TEST_P(OVInferRequestDynamicTests, InferDynamicNetwork) {
+    std::vector<ov::Shape> vectorShapes{{1, 4, 20, 20}, {1, 4, 20, 20}, {1, 4, 10, 10}};
+    bool checkOutputValues = !strcmp(function->get_friendly_name().c_str(), "Sin");
+    const std::string tensor_name = "input_tensor";
+    std::map<std::string, ov::PartialShape> shapes;
+    shapes[tensor_name] = { ov::Dimension(1, 5), ov::Dimension(1, 5), ov::Dimension(10, 40), ov::Dimension(10, 40) };
```

Comment:
looks like enough to pass as test parameter
`::testing::Values(getFunction3({{1,5}, {1, 5}, {10, 40}, {10, 40})),`